resources/RemoveInactiveUsersRunner.php

```diff
@@ -57,36 +71,60 @@
     if ($lastLoginDate) { // null lastLoginDate check
         $interval = $today->diff($lastLoginDate);
     } else { // This might only be run once, since new users always have field filled.
-        echo "User has no lastLoginDate (it may have been a very long time.)\n";
-        echo "Deleting user.\n";
+        echo "Deleting this user as it has no last login date " .
+        "(it may have been a very long time). \n";
+        if ($dryRun == true) {  // If dry run option has been selected
+             //writing to file to say who would have been deleted
+            file_put_contents("InactiveUsersToBeDeleted.txt", $user->getId() . ", `null`, " .
+            "`?`\n", FILE_APPEND);
+        };  // no else needed.
         deleteUser($user, $entityManager);
         echo "\n";
         // Move onto the next users.
         continue;
     }
 
-    $elapsedMonths = (int) $interval->format('%a') / 30;
-    echo 'Months elapsed since last login: ' . $elapsedMonths . "\n";
+    $elapsedMonths = (int) $interval->format('%a') / 30.4375; // 30.4375 is 365.25/12
+    // Rounded months since last login to 2 decimal places on line below for easier reading.
+    echo 'Months elapsed since last login: ' . round($elapsedMonths, 2) . "\n";
 
     if ($elapsedMonths > $deletionThreshold) { // Delete user
-        echo "Deleting user\n";
-        deleteUser($user, $entityManager);
+        // Check if it is a dry run
+        if ($dryRun == true) {
+            // Dry run option is set, so append user to be deleted to the file
+            echo "This user will be deleted when it isn't a dry run. \n\n";
```

Comment:
```suggestion
            echo "This user will be deleted when it isn't a dry run.\n";
```

One new line is probaby fine here, to match the output on a "Do Nothing".